tests/unit/adapters/network_importer/test_load_batfish_ipaddress.py

```diff
@@ -46,8 +48,16 @@ def test_load_batfish_ipaddress_wo_ip_wo_prefix(network_importer_base, site_sfo,
     prefix = Prefix(prefix="10.10.10.0/24", site_name="sfo")
 
     assert isinstance(ipaddr, IPAddress)
-    assert adapter.get(IPAddress, identifier=ipaddr.get_unique_id()) is None
-    assert adapter.get(Prefix, identifier=prefix.get_unique_id()) is None
+    try:
+        res = adapter.get(IPAddress, identifier=ipaddr.get_unique_id())
+    except ObjectNotFound:
+        res = None
+    assert res is None
```

Comment:
Simpler pattern for all of these test changes:

```python
with pytest.raises(ObjectNotFound):
    adapter.get(IPAddress, identifier=ipaddr.get_unique_id())
```

Reply:
Changes added